### ISSUE TITLE
(Report sent using device ID 966a4a38cdb3284d and IP address 23.161.247.103 with status code 404 ✅)

### ISSUE THREAD

**Issue by darkzone964** [qorsan73/Fuck-feca#1]
all report its just like this 404 ?

**Comment by qorsan73** [comment]
Yes, it is being reported, but if the account does not violate Facebook's terms of service, the account will not be deleted

**Comment by Zengg987** [comment]
its mean the tool doesnt work bruh 404 mean the request not send there is such url with "report" on facebook this tool totaly ass
